app/components/Utility/BindToChainState.jsx

```diff
@@ -436,7 +459,29 @@ function BindToChainState(Component, options = {}) {
                     if (this.state[key]) new_state[key] = null;
                 }
             }
-
+            /* Resolve assets */
+            for (let key of this.chain_liquidity_pools) {
+                let prop =
+                    props[key] ||
+                    this.dynamic_props[key] ||
+                    this.default_props[key];
+                // console.log("-- Wrapper.update, chain_liquidity_pools -->", key, prop);
+                if (prop) {
+                    const pools = await ChainStore.getLiquidityPoolsByShareAsset(
```

Comment:
This breaks pattern of the way of fetching here. It might be preferrable, but it needs to be changed separately and for all caches. Also, this may delay the renderer

Reply:
Hm?